rts/rts.cabal

```
@@ -270,6 +511,7 @@ library
          cpp-options: -DNOSMP
 
       include-dirs: include
+      include-dirs: .
```

Oof, so we include headers from `.` _and_ `include`? I guess we want a 
```suggestion
      -- TODO: move headers into `include`.
      include-dirs: .
```

---

Yes, the ones in `include` are the public ones (RtsApi.h, etc.) while the ones at the root are just to build the rts C code.

---

If we split private/public maybe we should have `include`, and `include/private`? (even if we publish them, people _could_ include them if they wanted to, but it's clearly marked private)?

---

I've added a todo